README.md

```diff
@@ -1,18 +1,11 @@
-# ClusterLink Project
+# ClusterLink
 
-## Disclaimers and Warnings
```

Comment:
I think we should keep the caveat for quite some time still. It can be reworded, but not removed